### ISSUE TITLE
Many wrong "Unused global declaration" 's

### ISSUE THREAD

**Issue by binarykitchen** [cursive-ide/cursive#2008]
In latest IntelliJ + Cursive I suddenly see lots of `Unused global declaration` warnings which weren't there yesterday.

For example 

```cljs
(defn initialize-callback <--- this is greyed out and Cursive thinks it's unused
  [reason]
  (logging/info (str "Initializing with reason " reason)))

(defn ^:export init
  []
  (office/initialize initialize-callback))
```

But when I add a random whitespace somewhere inside the initialize-callback function, this warnings disappears.

Refresh, caching issue maybe?

**Comment by cursive-ide** [comment]
Yes, sometimes the indices can get corrupted. Does *File->Invalidate Caches and Restart* fix this?

**Comment by binarykitchen** [comment]
No already tried multiple times and doesn't help.

**Comment by cursive-ide** [comment]
Can you attach a screenshot of what you're seeing?

**Comment by binarykitchen** [comment]
![screen shot 2018-07-12 at 12 15 44](https://user-images.githubusercontent.com/177641/42605661-62136ccc-85cd-11e8-8410-e8ce280a6bb6.png)

**Comment by cursive-ide** [comment]
Is that CLJS or CLJC?

**Comment by binarykitchen** [comment]
CLJS

**Comment by mikew1** [comment]
Same issue here.  As to possible specific cause, I have some repetitions of the same 'def' in my file, like `def user-data` appears more than once.  This is not an error in my case as I am working incrementally.  This results in various incorrect "unused global declarations", not limited to the ones I define more than once.
